roles/realm_join/files/httpd-gssproxy.service

```diff
@@ -0,0 +1,3 @@
+.include /lib/systemd/system/httpd.service
```

Comment:
`.include` isn't needed if you create it as a drop in. That means storing it as `/etc/systemd/system/httpd.service.d/gss-use-proxy.conf` (or similar). On newer systemd versions (included in EL8) the `.include` statement generates a deprecation warning.

Reply:
I'm trying to follow the Satellite manual as strictly as possible - that is why I'm having the `gssproxy` steps even though those are not used.
Noted for the manual improvements tho! Thanks!

Reply:
We should file a docs bug then.

Reply:
I'll followup with the improvements of the feature, docs improvement is part of it, I will take this into account :+1:

Reply:
I still don't like implementing this because it's not the way it's supposed to be used. By placing the file in `/etc/systemd/system/httpd.service` it means you need to have a migration in the future. Probably not a huge issue for most forklift deployments, but I did want to mention it.